projects/sandbox/train/train.py

```diff
@@ -134,6 +130,8 @@ def main(
     preprocessor.normalizer.fit(intrinsic)
     preprocessor.normalizer.to(device)
 
+    torch.save(preprocessor, outdir / "preprocessor.pt")
```

Comment:
How do we re-use this?

Reply:
I *think* we'll be able to call preprocessor.load_state_dict(). Should double check that

Reply:
Fixed this so that we save the `whitener` and `scaler` individually